atlas/rules/genecatalog.snakefile

```diff
@@ -64,7 +64,7 @@ if (config['genecatalog']['clustermethod']=='linclust') or (config['genecatalog'
             faa= "Genecatalog/all_genes/predicted_genes.faa"
         output:
             db=temp(directory("Genecatalog/all_genes/predicted_genes")),
-            clusterdb = directory("Genecatalog/clustering/mmseqs"),
+            clusterdb = temp(directory("Genecatalog/clustering/mmseqs"))
```

Comment:
@SilasK This directory does not seem necessary for the user. Okay to make a temp directory and delete once ATLAS is done?